### ISSUE TITLE
Can't apply patch to Wii U VC rom via the site

### ISSUE THREAD

**Issue by doodleintime** [tewtal/sm_practice_hack#91]
By using a Wii U homebrew app named dumpling and using a PC tool called wiiu-vc-extractor, you can extract a Super Metroid rom from the Virtual Console release of Super Metroid that runs in emulators, and you can even apply rom hack patches to it and those will work as well. 

The thing is, though, that the VC rom has some subtle differences compared to the original SNES release of Super Metroid, which means it has a different checksum, so trying to patch the practice hack onto the VC rom via the site results in a "The select file's checksum does not match the expected value, try with another file." error. Is there any chance that the checksum for the VC rom could be added to the site, or does the practice hack have issues with the VC rom?

**Comment by doodleintime** [comment]
Update: I compiled the ips patches and applied the non-sd2snes patch to the the VC rom, and it seems to work fine.

**Comment by idlechild** [comment]
I agree the practice rom should be compatible with the VC rom.  It looks like the only things Nintendo changed were palettes.  I documented it here:
https://github.com/InsaneFirebat/sm_practice_hack/pull/5/commits/dcf34c209a68230210ecf73b8f8d266752894a09

I got my information from here:
https://www.romhacking.net/forum/index.php?topic=25461.msg349133#msg349133

**Comment by idlechild** [comment]
So the request is to accept either the original NTSC/JU checksum or the VC NTSC/JU checksum as a valid NTSC rom.  I assume that would not be difficult to implement.  However I will defer to @tewtal since it is his website.  There might be a reason why it is not supported.

**Comment by InsaneFirebat** [comment]
It should be safe to add more checksums to the list. total mentioned using it to add romhack support where it made sense. We should probably do that for SM Impossible since it has no known conflicts, and zoast has been using it for a while now.

**Comment by idlechild** [comment]
I thought SMI + vanilla practice rom had problems in crocomire's room.  Also it's possible that the area rando changes messed up more compatibility.

Of course it'd fine (more work but fine) to have separate tabs for the various ROM hacks with GitHub source or branch for each hack, but I don't think offering up a few hacks with vanilla practice rom is a good idea.

**Comment by tewtal** [comment]
There's no way currently to have multiple accepted checksums for a ROM, but it wouldn't be too hard to add support for it. I would need to have the CRC32 checksum for the Wii VC ROM though to be able to implement it.
